### ISSUE TITLE
makefile issue - mkdir

### ISSUE THREAD

**Issue by dimitrisdovinos** [ruby/racc#147]
The makefile did not work for my setup (Ubuntu 20)

make "DESTDIR=" install
make: /usr/bin/mkdir: Command not found

I had to manually add a symlink to mkdir
sudo ln -s /bin/mkdir /usr/bin/mkdir

**Comment by withhawaii** [comment]
I had the same issue on Ubuntu 20.04 with Ruby 2.6.6.

@dimitrisdovinos your fix worked like a charm.  
Thank you so much!

**Comment by sbocinec** [comment]
@dimitrisdovinos @withhawaii I guess this is caused by using `rvm` version manager and it's not a `racc` project issue. 

Instead of creating a symlink I rather recommend to recompile the installed ruby rvm version with `--disable-binary` as the issue is most probably caused by the precompiled binary ruby release the rvm installs. This should also fix other potential issues the binary version might have introduced.

TL;DR:
Reinstall ruby version from source to fix the issue: `rvm reinstall <ruby-version> --disable-binary`.
Then `racc` gem installation should succeed.

Further notes:
* I think this issue should be closed
* more info in a related `ffi` bug https://github.com/ffi/ffi/issues/822

**Comment by gaganawhad** [comment]
Thanks for creating this issue @dimitrisdovinos. @sbocinec seems more informed, so I am going to follow his suggestion and reinstall ruby.

**Comment by hsbt** [comment]
@sbocinec Thanks for explanation ❤️
